config/src/projects/optimism.ts

```diff
@@ -5,15 +5,32 @@ export const optimism: Project = {
   slug: 'optimism',
   bridges: [
     {
+      // TODO: this one seems unused but there are tokens locked
       address: '0x045e507925d2e05D114534D0810a1abD94aca8d6',
       sinceBlock: 11656238,
       tokens: ['SNX'],
     },
     {
+      // TODO: this one seems unused but there are tokens locked
       address: '0x5Fd79D46EBA7F351fe49BFF9E87cdeA6c821eF9f',
       sinceBlock: 12409015,
       tokens: ['SNX'],
     },
+    {
+      address: '0xCd9D4988C0AE61887B075bA77f08cbFAd2b65068',
+      sinceBlock: 12409013,
+      tokens: ['SNX'],
+    },
+    {
+      address: '0x10E6593CDda8c58a1d0f14C5164B376352a55f2F',
```

Comment:
This should be: `0x467194771dAe2967Aef3ECbEDD3Bf9a310C76C65` (funds are escrowed outside of a bridge contract).

Reply:
Solved via #115